test/poly/PolyRing-test.jl

```diff
@@ -24,37 +24,40 @@ end
    I.isGB = true
    S = fres(I, 0)
    M = S[2]
-   @test string(M[1]) == "x*gen(2)-y*gen(1)+z*gen(2)-z*gen(1)"
+   ss = string.(Singular.singular_symbols(R))
+   @test string(M[1]) == ss[1]*"*gen(2)-"*ss[2]*"*gen(1)+"*ss[3]*"*gen(2)-"*ss[3]*"*gen(1)"
 
    R, (x, y, z) = polynomial_ring(QQ, ["x", "y", "z"], ordering2 = :comp1max)
    I = Ideal(R, x+z, y+z)
    I.isGB = true
    S = fres(I, 0)
    M = S[2]
-   @test string(M[1]) == "x*gen(2)-y*gen(1)-z*gen(1)+z*gen(2)"
+   ss = string.(Singular.singular_symbols(R))
+   @test string(M[1]) == ss[1]*"*gen(2)-"*ss[2]*"*gen(1)-"*ss[3]*"*gen(1)+"*ss[3]*"*gen(2)"
 
    R, (x, y, z) = polynomial_ring(QQ, ["x", "y", "z"], ordering = :comp1min,
          ordering2 = :degrevlex)
    I = Ideal(R, x+z, y+z)
    I.isGB = true
    S = fres(I, 0)
    M = S[2]
-   @test string(M[1]) == "x*gen(2)+z*gen(2)-y*gen(1)-z*gen(1)"
+   ss = string.(Singular.singular_symbols(R))
+   @test string(M[1]) == ss[1]*"*gen(2)+"*ss[3]*"*gen(2)-"*ss[2]*"*gen(1)-"*ss[3]*"*gen(1)"
 
    R, (x, y, z) = polynomial_ring(QQ, ["x", "y", "z"], ordering = :comp1max,
          ordering2 = :degrevlex)
    I = Ideal(R, x+z, y+z)
    I.isGB = true
    S = fres(I, 0)
    M = S[2]
-   @test string(M[1]) == "[-y-z,x+z]"
+   ss = string.(Singular.singular_symbols(R))
+   @test string(M[1]) == "[-"*ss[2]*"-"*ss[3]*","*ss[1]*"+"*ss[3]*"]"
 end
 
 @testset "PolyRing.ordering.fancy" begin
    function test_monomials(l)
       for i in 2:length(l)
          @test l[i-1] > l[i]
-         @test l[i] < l[rand(1:i-1)]
```

Comment:
Why was this removed?

Reply:
Hmm, as near as I can tell, we check that each `l[i] < l[i-1]`, then also that `l[i] < l[j]` for some random `j in 1:i-1`. It seems this second test is redundant, or am I missing something? I can add this one back, no problem.

Reply:
(as for why I messed with it in this pull request, I needed to fix the other ordering tests because of the variable names, and was looking at this one to compare.)